CHANGELOG_PENDING.md

```diff
@@ -11,3 +11,5 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 ### IMPROVEMENTS:
 
 ### BUG FIXES:
+
+- [blockchain] \#5249 Fix fast sync halt with initial height > 1 (@erikgrinaker)
```

Comment:
Does this need a changelog entry if its fixing something that hasn't been released, not rc? This subject was touched on in the tm-rs talk. Do we have an approach to it?

Reply:
Well, it fixes a bug in rc2, so presumably the people who are testing rc2 would like to know that it's been fixed in rc3.

Reply:
so it wouldn't land in the 34.0 changlog or it would?

Reply:
Depends on what our policy is. Some projects only include the changes between the last RC and the final release (this is also what I've been doing for IAVL), others include all changes from the last stable release (which means changes would be listed twice; once in the RCs and once for the stable release). I don't have a strong opinion.

Reply:
likewise, I never put much thought to it until bucky brought it up in the tm-rs call. It's not an urgent thing, so we can discuss with the rest of the team if they have a preference.